Fabulous.XamarinForms/src/Fabulous.Android/Host.fs

```diff
@@ -0,0 +1,29 @@
+namespace Fabulous.Android
+
+open Android.App
+open Fabulous
+
+
+type IAndroidHost =
+    abstract Context: Android.Content.Context
+    inherit IHost
+
+type AndroidContextHost(context) =
```

Comment:
This code used only to instaniate items inside ViewGroup to pass Context of ViewGroup to .Create function